src/component/Board.jsx

```diff
@@ -62,22 +66,35 @@ function Board() {
     setMovesCounter(0);
   };
 
+  const scoreCalculation = () => {
+    baseScore = cardLevel / 2 - 1;
+    score = Math.round((baseScore / movesCounter) * 100);
+  };
+
   // function to check game is ended or not
   const gameEnd = () => {
+    scoreCalculation();
```

Comment:
Instead of storing it can we return from here & show it on UI?

Reply:
implemented